src/database/redis.js

```diff
@@ -12,7 +13,7 @@ redisModule.questions = [
 	{
 		name: 'redis:host',
 		description: 'Host IP or address of your Redis instance',
-		default: nconf.get('redis:host') || '127.0.0.1',
+		default: nconf.get('redis:host') || ip.address(),
```

Comment:
The default should be to assume the database is installed on the local machine, and to use the shorthand `127.0.0.1`. If the default were the external IP of the server, and the installation was moved, the config would no longer be correct.